src/app/forms/templates/general/applicant-details.template.ts

```diff
@@ -43,7 +43,7 @@ export const ApplicantDetails: FormNode = {
       customId: 'applicantPostTown'
     },
     {
-      name: 'techRecord_purchaserDetails_address3',
+      name: 'techRecord_applicantDetails_address3',
```

Comment:
why was this wrong in the first place? Just to make sure you're not changing to match the `cvs-type-definitions` if it's wrong.